spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/writer/WriteConf.scala

```diff
@@ -203,6 +212,13 @@ object WriteConf {
 
     val ttlOption = if(ttlSeconds == TTLParam.default) TTLOption.defaultValue else TTLOption.constant(ttlSeconds)
 
+
+    val timestampMillis = conf.getLong(TimestampParam.name, TimestampParam.default)
+
+    val timestampOption =
```

Comment:
I believe our style on this is if you have to break multiple lines
http://docs.scala-lang.org/style/control-structures.html
```
If (something)
  do this
else
  do that
```